### ISSUE TITLE
Finishing the website + ADA compliance

### ISSUE THREAD

**Issue by omartounsi7** [omartounsi7/ECE461_Part2#93]
The HTML should be updated so that our website has the latest features.

**Comment by omartounsi7** [comment]
And while we're at it, we should make sure our website passes these ADA compliance tests:

https://www.accessibilitychecker.org/

**Comment by omartounsi7** [comment]
Website must be ADA compliant via WCAG 2.1 at level AA
